ui/jobs/buff_tracker.ts

```diff
@@ -5,6 +5,33 @@ import EffectId from '../../resources/effect_id';
 import { MatchesAbility, MatchesGainsEffect, MatchesLosesEffect } from '../../resources/matches';
 import PartyTracker from '../../resources/party';
 
+import potionImage from '../../resources/ffxiv/status/potion.png';
+import astralImage from '../../resources/ffxiv/status/astral.png';
+import umbralImage from '../../resources/ffxiv/status/umbral.png';
+import physicalImage from '../../resources/ffxiv/status/physical.png';
+import offguardImage from '../../resources/ffxiv/status/offguard.png';
+import peculiarLightImage from '../../resources/ffxiv/status/peculiar-light.png';
+import trickAttackImage from '../../resources/ffxiv/status/trick-attack.png';
+import battleLitanyImage from '../../resources/ffxiv/status/battle-litany.png';
+import emboldenImage from '../../resources/ffxiv/status/embolden.png';
+import arrowImage from '../../resources/ffxiv/status/arrow.png';
+import balanceImage from '../../resources/ffxiv/status/balance.png';
+import boleImage from '../../resources/ffxiv/status/bole.png';
+import ewerImage from '../../resources/ffxiv/status/ewer.png';
+import spearImage from '../../resources/ffxiv/status/spear.png';
+import spireImage from '../../resources/ffxiv/status/spire.png';
+import ladyOfCrownsImage from '../../resources/ffxiv/status/lady-of-crowns.png';
+import lordOfCrownsImage from '../../resources/ffxiv/status/lord-of-crowns.png';
+import devilmentImage from '../../resources/ffxiv/status/devilment.png';
+import standardFinishImage from '../../resources/ffxiv/status/standard-finish.png';
+import technicalFinishImage from '../../resources/ffxiv/status/technical-finish.png';
+import battlevoiceImage from '../../resources/ffxiv/status/battlevoice.png';
```

Comment:
Sorry, this is the tiniest nit, but can this be `battleVoiceImage`? Battle Voice is two words.  <_<